client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -113,17 +159,27 @@ module.exports = React.createClass( {
 					volume: cartItem.volume,
 					productName: cartItem.product_name
 				}
-			};
-
+			},
+			hasDomainSuffix, gappsProductName;
+		if ( isDomainRegistration( cartItem ) ) {
+			return cartItem.product_name;
+		}
 		if ( ! cartItem.volume ) {
 			return cartItem.product_name;
 		} else if ( cartItem.volume === 1 ) {
 			switch ( cartItem.product_slug ) {
 				case 'gapps':
+					hasDomainSuffix = cartItem.product_name.indexOf( cartItem.meta ) > -1;
+					gappsProductName = hasDomainSuffix ? cartItem.product_name : this.translate( '%(productName)s for %(domain)s', {
```

Comment:
Could we move the string literal to the new line? It would be easier to read here